connectors/sources/confluence.py

```diff
@@ -204,6 +235,39 @@ async def _handle_client_errors(self, url, exception):
         else:
             raise
 
+    async def _handle_client_payload_error(self, exception):
+        retry_seconds = DEFAULT_RETRY_SECONDS
+        response_headers = exception.headers or {}
+        if "Retry-After" in response_headers:
+            try:
+                retry_seconds = int(response_headers["Retry-After"])
+            except (TypeError, ValueError) as exception:
+                self._logger.error(
+                    f"Error while reading value of retry-after header {exception}. Using default retry time: {DEFAULT_RETRY_SECONDS} seconds"
+                )
+        await self._sleeps.sleep(retry_seconds)
+        raise
+
+    async def _handle_api_call_error(self, url, exception):
+        known_errors = {
+            ServerConnectionError,
+            ClientResponseError,
+            ClientPayloadError,
```

Comment:
Why is this in the list if there's a `_handle_client_payload_errror()` function?

Reply:
To re-raised the exception if error is not handled by `_handle_client_payload_errror()`